.github/workflows/docs_deploy.yml

```diff
@@ -0,0 +1,70 @@
+name: Deploy Documentation
+
+on:
+  push:
```

Comment:
how does this work exactly? 

Let's say that we push a `tag` (that's how we cut a release today), to release the docs, does the `paths` line mean that the change should also include a change in the `docs/` directory?


if the change was in a previous commit, does it mean that this deployment will get skipped?